Unknown CMake command "check_symbol_exists".

Hello.

I have just git cloned this URL right now:

https://github.com/hyperrealm/libconfig

Now I am trying to build it but cmake tells me this error:

-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /System/Index/bin/c++
-- Check for working CXX compiler: /System/Index/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at lib/CMakeLists.txt:47 (check_symbol_exists):
  Unknown CMake command "check_symbol_exists".


-- Configuring incomplete, errors occurred!
See also "/Depot/jjjj/libconfig/CMakeFiles/CMakeOutput.log".




My host system is linux; g++ version and gcc version is 8.2.0;
my cmake version is cmake version 3.13.3.

No clue where the error is but hopefully you guys can figure
it out. I compiled both g++ (gcc),  cmake, binutils etc... from
source, either the most recent tarball release, or also from 
git checkout.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,6 +23,7 @@ else()
 endif()
 
 include(GNUInstallDirs)
+include(CheckSymbolExists)
 add_subdirectory(lib)
 
 if(BUILD_EXAMPLES)

